robert_beverage/scripts/web_server

```diff
@@ -1,5 +1,30 @@
 #!/usr/bin/env python
 import rospy
+import socket
+import SocketServer
+import SimpleHTTPServer
 
 rospy.init_node('web_server_node')
-rospy.loginfo('Inside web_server script')
\ No newline at end of file
+rospy.loginfo('Inside web_server script')
+
+HOST = socket.gethostname()
+PORT = 8080
+address = ("", PORT)
+
+
+class RequestHandler(SimpleHTTPServer.SimpleHTTPRequestHandler):
+    def do_GET(self):
+        if self.path.startswith("/beverage"):
+            info = self.path.strip("/")[9:]
+            x, y = info.split("&")
+            print "{}\n{}".format(x, y)
```

Comment:
please do not use prints but ROS logging instead http://wiki.ros.org/rospy/Overview/Logging